pkg/core/mempool/mem_pool_test.go

```diff
@@ -256,15 +246,6 @@ func TestOverCapacity(t *testing.T) {
 	// But claim tx should still be there.
 	require.True(t, mp.ContainsKey(claim.Hash()))
 
-	// Low net fee, but higher per-byte fee is still a better combination.
```

Comment:
Not sure why this one is deleted.

Reply:
That was after I removed `FeePerByte` from blockchain. Added it again now.

Reply:
Now it's being moved for no good reason.

Reply:
Moved this test to its original place and realised that it fails. The reason is in transactions' comparison: now we calculate `feePerByte` using *transactions'* network fee (not the feer's one), so we can't have the case were transaction has low network fee and higher `feePerByte`. `feer.feePerByte` does not affects the comparison in this case.

Reply:
>  we can't have the case were transaction has low network fee and higher feePerByte

Why? It just depends on transaction size.

Reply:
I mean, for transactions with the same size. Here we already have empty contract txes in mempool, so another one should be smaller then the others to fit. Can I add some attributes to previous transactions to make them bigger?

Reply:
That's one option, another one of course is raising the fee for it.